tools/c7n_azure/tests_azure/container-host/test_container_host.sh

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env bash
+
+set -e
+export AZURE_CORE_OUTPUT="none"
+
+resource_group_name="custodian-container-host-nightly"
+rg_tests=("aci-periodic" "aci-event" "aks-periodic" "aks-event")
+
+function cleanup {
+    set +e
+    echo "################################### Cleaning Up ###################################################"
+    echo "Deleting all test resources"
+    for rg_test in "${rg_tests[@]}"; do
+        az group delete --name "container-host-nightly-$rg_test" --yes
+    done
+    az group delete --name $resource_group_name --yes
+    unset AZURE_CORE_OUTPUT
+}
+trap cleanup EXIT
+
+echo "Logging in to Azure"
+az login --service-principal --username $AZURE_CLIENT_ID --password $AZURE_CLIENT_SECRET --tenant $AZURE_TENANT_ID
+az account set --subscription $AZURE_SUBSCRIPTION_ID
+
+echo "################################## Creating Shared Resources #######################################"
+
+# Resource Group
+echo "Creating Resource Group"
+location="westus2"
+az group create --name $resource_group_name --location $location
+
+# Storage Account
+echo "Creating Storage Account"
+storage_account_name="c7ncontainernightly"
+az storage account create --resource-group $resource_group_name --name $storage_account_name
+storage_account_key=$(az storage account keys list --account-name $storage_account_name --query "[0].value" --output tsv)
+read -r storage_account_id storage_blob_endpoint <<<$(az storage account show --name $storage_account_name --query "{id: id, blobEndpoint: primaryEndpoints.blob}" --output tsv)
+
+# Application Insights
+echo "Creating Application Insights"
+app_insights_name="custodian-insights"
```

Comment:
nit: I think it would be nice to move all hardcoded variables to the top, so it is easy to see what is configurable..